test/witan/send/send_test.clj

```diff
@@ -73,10 +73,33 @@
                        :Non-SEND :Non-SEND :Non-SEND :Non-SEND :Non-SEND :Non-SEND
                        :Non-SEND :Non-SEND :Non-SEND :Non-SEND :Non-SEND :Non-SEND
                        :Non-SEND :Non-SEND :Non-SEND :Non-SEND]
-               :id [5 5 6 6 7 7 8 8 9 9 10 10 11 11 12 12 13 13 14 14 15 15 16 16
-                    17 17 18 18 19 19 20 20 21 21 22 22 23 23 24 24]
                :sim-num [1 2 1 2 1 2 1 2 1 2 1 2 1 2 1 2 1 2 1 2 1 2 1 2 1 2 1 2 1 2
-                         1 2 1 2 1 2 1 2 1 2]}))
+                         1 2 1 2 1 2 1 2 1 2]
+               :id [5 5 6 6 7 7 8 8 9 9 10 10 11 11 12 12 13 13 14 14 15 15 16 16
+                    17 17 18 18 19 19 20 20 21 21 22 22 23 23 24 24]}))
+
+(def population-total
+  (ds/dataset {:year [2014 2014 2014 2014 2014 2014 2016 2016
```

Comment:
Having columns laid out like this makes it almost impossible to parse a single row, which obviously makes changes hairy and fragile. As a CSV it'd be way easier to grok. Also, in other projects we've loaded CSVs.